charts/growthbook/templates/ingress.yaml

```diff
@@ -0,0 +1,35 @@
+{{- if .Values.ingress.enabled -}}
```

Comment:
we do not use ingreses from charts since our clusters have differing versions one ingress apiVersion won't work for all clusters. So you can keep it here but ensure that the default enabled value is false.

We have another repo for Ingreses

Reply:
Yes, by default ingress is disabled. PR specific for ingress : https://github.com/one-acre-fund/k8s-ingresses/pull/160